spec/unit/controllers/v3/apps_controller_spec.rb

```diff
@@ -1489,6 +1489,81 @@
     end
   end
 
+  describe '#clear_buildpack_cache' do 
+    let(:app_model) { VCAP::CloudController::AppModel.make(droplet_guid: droplet.guid, desired_state: 'STARTED') }
+    let(:droplet) { VCAP::CloudController::DropletModel.make(state: VCAP::CloudController::DropletModel::STAGED_STATE) }
+    let(:space) { app_model.space }
+    let(:org) { space.organization }
+    let(:user) { VCAP::CloudController::User.make }
+
+    before do
+      set_current_user(user)
+      allow_user_read_access_for(user, spaces: [space])
+      allow_user_write_access(user, space:)
+      VCAP::CloudController::BuildpackLifecycleDataModel.make(app: app_model, buildpacks: nil, stack: VCAP::CloudController::Stack.default.name)
+    end
+
+    it 'returns a 200 and the app' do
+      put :clear_buildpack_cache, params: { guid: app_model.guid }, as: :json
```

Comment:
I wonder why `put` is used here and also in other tests e.g. for :start, :restart and so on. Shouldn't we use `post`?